### PR TITLE
allow github URL to be more easily changed #415

### PR DESCRIPTION
**Pull request by chris18890**
allow github URL to be more easily changed #415

### CODE REVIEW

**Comment by chris18890** [comment]
@ramgarden @elplatt any objections to this being merged?

File: crm/themes/inspire/email-welcome.tpl.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
     print 'Welcome to ' . "$config_org_name" . '!';
     print '<br /><br />';
     
-    print 'You are receiving this email because you have recently been entered into the ' . "$config_org_name" . ' membership management system, ' . "$config_site_title" . '.';
+    print 'You are receiving this email because you have recently been entered into the ' . "$config_org_name" . ' user management system, ' . "$config_site_title" . '.';
```
Comment (ramgarden) [inline crm/themes/inspire/email-welcome.tpl.php]: Are we really changing it to be a user management system? I thought it was crm for makerspace members? And other groups members...

**Comment by chris18890** [comment]
@elplatt @ramgarden removed the change to the joining email, happy for this to be merged now?

**Comment by ramgarden** [comment]
I don't remember what it was before.  But this set of changes looks OK.

**Comment by chris18890** [comment]
if you click on the outdated diff button above you can see that I changed the wording in the email from membership to user, have reverted it now though so will merge :)

**Comment by ramgarden** [comment]
TIL about the outdated diff button.  Yep now I see and it looks good to go.

**Pull request merged**